app/models/hood_difference.rb

```diff
@@ -0,0 +1,82 @@
+  class HoodDifference
```

Comment:
I feel like this class doesn't belong under `app/models`.
What made you put it here?

Reply:
I'm open to suggestions here!

I was thinking about what this class is and I got kinda baffled. This COULD become a `struct` and we can have a `app/structs` folder, but because it has state + methods I wasn't sure that this would be suitable for a struct.

This is a PORO (plain old ruby object), which I generally don't have any idea where to place. I thought about creating a `app/domain` folder so that we can keep our domain models (which this [MIGHT qualify as a domain model](https://en.wikipedia.org/wiki/Domain_model)).

In a few tutorials on POROs, I noticed Rails people put them in the `app/models` directory. Examples here: 

- https://www.codewithjason.com/extracting-tidy-poro-messy-active-record-model/ (see comments)
- https://www.toptal.com/ruby-on-rails/decoupling-rails-components

Reply:
Hm I like `app/domain` that sounds interesting but it sounds like there's already a convention of people leaving POROs in `app/models` so I'm fine with this - it's a non-blocker.